src/PowerShell/Microsoft.WinGet.DSC/Microsoft.WinGet.DSC.psm1

```diff
@@ -52,6 +52,12 @@ enum WinGetTrustLevel
     Trusted
 }
 
+enum WinGetScope

```

Comment:
Seems like this needs to be a subset of `EnvironmentVariableTarget`. I didn't know you could use enums like that

Small nit: if it is related to the other enum, I would add a comment for the future